internal/edgedbtypes/numbers.go

```diff
@@ -32,6 +32,9 @@ func (o *Optional) Missing() bool { return !o.isSet }
 // means present.
 func (o *Optional) SetMissing(missing bool) { o.isSet = !missing }
 
+// Unset marks the value a missing
+func (o *Optional) Unset() { o.isSet = false }
```

Comment:
Why do we need `Unset` in addition to `SetMissing`?

Reply:
Because the new query single behavior uses `type unseter interface { Unset() }` to identify optional result types. I was thinking about removing `SetMissing(bool)` and replacing it with `Set()`, but I'm not sure.